util/misc.go

```diff
@@ -531,3 +543,62 @@ func GetLocalIP() string {
 	}
 	return ""
 }
+
+func createTLSCertificates() error {
+
```

Comment:
Stylistically we don't normally add a space here (after function definition or before closing `}`). Reviewers are normally more relaxed in tests though.